gradle/libs.versions.toml

```diff
@@ -6,6 +6,10 @@ composeVersion = "1.6.11"
 okhttp = "4.9.2"
 springTwo = "2.7.18"
 springThree = "3.4.2"
+targetSdk = "34"
```

Comment:
There’s no way to have aliases or have `targetSdk` and `compileSdk` values reference each other.